### PR TITLE
Added Debugs to all of Kubernetes.cljs

### PR DESCRIPTION
**Pull request by nsauce**

### CODE REVIEW

File: src/degree9/documentdb.cljs [hunk 1]
```diff
@@ -4,8 +4,11 @@
   [goog.object :as obj]
   [meta.promise :as p]
   [feathers.services :as svc]
+  [degree9.debug :as dbg]
   degree9.env
-  ["documentdb" :as documentdb]))
+  ["documentdb" :as documentdb]
+
+  (def ^:private debug (dbg "degree9:enterprise:documentdb"))))
```
Comment (flyboarder) [inline src/degree9/documentdb.cljs]: remove extra whitespace

File: src/degree9/hooks.cljs [hunk 2]
```diff
@@ -1,5 +1,8 @@
 (ns degree9.hooks
-  (:require [goog.object :as obj]))
+  (:require [goog.object :as obj]
+            [degree9.debug :as dbg]))
+
+(def ^:private debug (dbg "degree9:enterprise:server"))
```
Comment (flyboarder) [inline src/degree9/hooks.cljs]: wrong debug statement

File: src/degree9/idle.cljs [hunk 3]
```diff
@@ -1,5 +1,8 @@
 (ns degree9.idle
-  (:require [javelin.core :as j]))
+  (:require [javelin.core :as j]
+            [degree9.debug :as dbg]))
+
+(def ^:private debug (dbg "degree9:enterprise:server"))
```
Comment (flyboarder) [inline src/degree9/idle.cljs]: incorrect debug statement

File: src/degree9/kubernetes.cljs [hunk 4]
```diff
@@ -5,24 +5,41 @@
...
 
+
+(def ^:private debug (dbg "degree9:enterprise:kubernetes"))
+
 ;; Kubernetes API ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
-(defn kubeconfig []
+(defn kubeconfig
+  "Initializing the kubeconfig"
+  []
+  (debug "Initializing kubeconfig" conn)
```
Comment (flyboarder) [inline src/degree9/kubernetes.cljs]: remove `conn` variable

File: src/degree9/kubernetes.cljs [hunk 5]
```diff
@@ -5,24 +5,41 @@
...
   (k8s/KubeConfig.))
 
-(defn load-default [config]
+(defn load-default
+  "Loads the default configuration"
+  [config]
+  (debug "Loads default configuration" conn)
```
Comment (flyboarder) [inline src/degree9/kubernetes.cljs]: change `conn` to `config`

File: src/degree9/ping.cljs [hunk 6]
```diff
@@ -1,5 +1,8 @@
 (ns degree9.ping
-  (:require ["socket.io-client" :as io]))
+  (:require ["socket.io-client" :as io]
+            [degree9.debug :as dbg]))
+
+(def ^:private debug (dbg "degree9:enterprise:server"))
```
Comment (flyboarder) [inline src/degree9/ping.cljs]: incorrect statement

File: src/degree9/kubernetes.cljs [hunk 7]
```diff
@@ -5,24 +5,41 @@
...
 
-(defn core-api [config]
+(defn core-api
+  ""
+  [config]
   (mkclient config k8s/CoreV1Api))
 
-(defn apps-api [config]
+(defn apps-api
+  ""
```
Comment (flyboarder) [inline src/degree9/kubernetes.cljs]: add description

File: src/degree9/kubernetes.cljs [hunk 8]
```diff
@@ -59,12 +76,15 @@
...
   [k8s]
+  (debug "Converts kubernetes response to cljs" k8s)
   (js->clj k8s :keywordize-keys true))
 
 (defn- k8s-response [res]
   (obj/get res "body"))
 
-(defn k8s-error [err]
+(defn k8s-error
+  ""
```
Comment (flyboarder) [inline src/degree9/kubernetes.cljs]: add description

File: src/degree9/documentdb.cljs (hunk 1 above)
Comment (nsauce) [reply src/degree9/documentdb.cljs]: Got rid of whitespace

File: src/degree9/hooks.cljs (hunk 2 above)
Comment (nsauce) [reply src/degree9/hooks.cljs]: Change statement to "hooks" instead of server.

File: src/degree9/idle.cljs (hunk 3 above)
Comment (nsauce) [reply src/degree9/idle.cljs]: changed server to "idle" in statement

File: src/degree9/kubernetes.cljs (hunk 4 above)
Comment (nsauce) [reply src/degree9/kubernetes.cljs]: Removed the conn variable

File: src/degree9/ping.cljs (hunk 6 above)
Comment (nsauce) [reply src/degree9/ping.cljs]: Changed server to "ping" in statement

File: src/degree9/kubernetes.cljs (hunk 7 above)
Comment (nsauce) [reply src/degree9/kubernetes.cljs]: Added correct description

File: src/degree9/kubernetes.cljs (hunk 8 above)
Comment (nsauce) [reply src/degree9/kubernetes.cljs]: Added correct description

File: src/degree9/kubernetes.cljs (hunk 5 above)
Comment (nsauce) [reply src/degree9/kubernetes.cljs]: changed conn to config

File: src/degree9/kubernetes/service.cljs [hunk 9]
```diff
@@ -0,0 +1,83 @@
+(ns degree9.kubernetes.service
+  (:require [degree9.debug :as dbg]))
+
+(def ^:private debug (dbg "degree9:enterprise:kubernetes:service"))
```
Comment (flyboarder) [inline src/degree9/kubernetes/service.cljs]: update debug

File: src/degree9/hooks.cljs [hunk 10]
```diff
@@ -1,5 +1,8 @@
 (ns degree9.hooks
-  (:require [goog.object :as obj]))
+  (:require [goog.object :as obj]
+            [degree9.debug :as dbg]))
+
+((dbg/defdebug debug "degree9:enterprise:hooks"))
```
[the remark this answers is not in the document]
Comment (nsauce) [reply src/degree9/hooks.cljs]: extra bracket is now gone

File: src/degree9/kubernetes/custom_resource.cljs [hunk 11]
```diff
@@ -0,0 +1,169 @@
+(ns degree9.kubernetes.custom-resource
+  (:require [degree9.debug :as dbg]))
+
+(def ^:private debug (dbg "degree9:enterprise:kubernetes:custom-resource"))
```
[the remark this answers is not in the document]
Comment (nsauce) [reply src/degree9/kubernetes/custom_resource.cljs]: fixed debug to use debug macro

File: src/degree9/kubernetes/deployment.cljs [hunk 12]
```diff
@@ -0,0 +1,85 @@
+(ns degree9.kubernetes.deployment
+  (:require [degree9.debug :as dbg]))
+
+(def ^:private debug (dbg "degree9:enterprise:kubernetes:deployment"))
```
[the remark this answers is not in the document]
Comment (nsauce) [reply src/degree9/kubernetes/deployment.cljs]: Fixed debug

File: src/degree9/kubernetes/ingress.cljs [hunk 13]
```diff
@@ -0,0 +1,83 @@
+(ns degree9.kubernetes.ingress
+  (:require [degree9.debug :as dbg]))
+
+(def ^:private debug (dbg "degree9:enterprise:kubernetes:ingress"))
```
[the remark this answers is not in the document]
Comment (nsauce) [reply src/degree9/kubernetes/ingress.cljs]: Debug fixed

File: src/degree9/kubernetes/secret.cljs [hunk 14]
```diff
@@ -0,0 +1,84 @@
+(ns degree9.kubernetes.secret
+  (:require [degree9.debug :as dbg]))
+
+(def ^:private debug (dbg "degree9:enterprise:kubernetes:secret"))
```
[the remark this answers is not in the document]
Comment (nsauce) [reply src/degree9/kubernetes/secret.cljs]: Debug fixed

File: src/degree9/kubernetes/service.cljs (hunk 9 above)
Comment (nsauce) [reply src/degree9/kubernetes/service.cljs]: Debug fixed

File: src/degree9/kubernetes/namespace.cljs [hunk 15]
```diff
@@ -0,0 +1,79 @@
+(ns degree9.kubernetes.namespace
+  (:refer-clojure :exclude [namespace])
+  (:require [degree9.debug :as dbg]))
+
+(def ^:private debug (dbg "degree9:enterprise:kubernetes:namespace"))
```
[the remark this answers is not in the document]
Comment (nsauce) [reply src/degree9/kubernetes/namespace.cljs]: Debug fixed

**Pull request merged**